src/lib/schematics/utils/html.ts

```diff
@@ -10,6 +10,10 @@ import {Project} from './devkit-utils/config';
  * @param src the src path of the html file to parse
  */
 export function getHeadTag(host: Tree, src: string) {
+  if (!parse5) {
```

Comment:
Should this check be earlier? I want to make sure that we don't partially run the schematic before bailing out here.

Reply:
Done